data/base-sorting.yaml

```diff
@@ -0,0 +1,483 @@
+cambrinth:
+- .*cambrinth.*
```

Comment:
Some items can use cambrinth as a decoration. This is minimal concern, but it's another one of those "gotchas" by the flavor text team... :D

Reply:
Yeah, but what can we do? It's either remove cambrinth category or live with the gotchas

Reply:
The look charname can be set to false by default if you want.

Reply:
Yeah, my comment was more of a "you're going to see this come up" type of thing. I think sorting it is just fine, otherwise you'd need to then parse the string for type of cambrinth, would be messy.